fizzbuzz.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+
+int main()
+{
+    for(int i = 1; i < 101; i++) {
+        if (i % 3 && i % 5) {
```

Comment:
do you think putting `i%3` and `i%5` in variables will be more efficient? I wonder if the compiler optimizes it anyway.